Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -87,6 +76,17 @@
             "helpMarkDown": "Select type of web app to deploy. <br/> Note: Select Linux Web App for built-in platform images or custom container image deployments."
         },
         {
+            "name": "WebAppName",
+            "type": "pickList",
+            "label": "App Service name",
+            "defaultValue": "",
+            "required": true,
+            "properties": {
+                "EditableOptions": "True"
+            },
+            "helpMarkDown": "Enter or Select the name of an existing Azure App Service."
```

Comment:
@Ajay-MS , @RoopeshNair , should we include helpMarkdown saying "App services based on selected App type is only listed" (or any-other)?

Reply:
Yes, we should. Ensure the PE scenarios and Template scenarios are tested before the changes are checked-in.

Reply:
@sachinma - to review from PE perspective.